.env.example

```
@@ -60,4 +60,11 @@ AI_FLOWS_CONFIG='{"custom":{"success_flow":{"adapter":"flowise","predictionUrl":
 AI_ADAPTERS_CONFIG='{"flowise":{"secret":"test-no-real-secret"}, "n8n":{"apiKey":"test-n8n-api-key"}}'
 
 # Subscriptions
-ENABLE_SUBSCRIPTIONS=true
\ No newline at end of file
+ENABLE_SUBSCRIPTIONS=true
+
+# NATS Configuration
+NATS_URL=nats://localhost:4222
+NATS_TOKEN=nats_auth_token
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NATS_TOKEN key should go before the NATS_URL key

(UnorderedKey)